test/unit/ngsiv2/HTTP_reveice_measures-test.js

```diff
@@ -254,6 +329,79 @@ describe('HTTP: Measure reception ', function() {
                 }
             };
 
+        beforeEach(function(done) {
+            nock.cleanAll();
+            // This mock does not check the payload since the aim of the test is not to verify
+            // device provisioning functionality. Appropriate verification is done in tests under
+            // provisioning folder of iotagent-node-lib
+            contextBrokerMock = nock('http://192.168.1.1:1026')
+                .matchHeader('fiware-service', 'smartGondor')
+                .matchHeader('fiware-servicepath', '/gardens')
+                .post('/v2/entities?options=upsert')
+                .reply(204);
+
+            contextBrokerMock
+                .matchHeader('fiware-service', 'smartGondor')
+                .matchHeader('fiware-servicepath', '/gardens')
+                .post(
+                    '/v2/entities/e0130101/attrs',
+                    utils.readExampleFile('./test/unit/ngsiv2/contextRequests/timeInstantMeasures2.json')
+                )
+                .query({ type: 'sensor' })
+                .reply(204);
+
+            iotaJson.stop(function() {
+                config.iota.timestamp = true;
+                config.compressTimestamp = false;
+                iotaJson.start(config, function() {
+                    request(provisionOptions, function(error, response, body) {
+                        done();
+                    });
+                });
+            });
+        });
+
+        afterEach(function() {
+            config.iota.timestamp = false;
+            config.compressTimestamp = true;
+        });
+
+        it('should send its value to the Context Broker', function(done) {
+            request(optionsMeasure, function(error, result, body) {
+                contextBrokerMock.done();
+                done();
+            });
+        });
+    });
+
+    describe('When a POST measure arrives with a TimeInstant query parameter in the body for a device with explicitAttrs:true', function() {
```

Comment:
Similar to a previous comment, for simmetry, the describe in line 304 should be changed to:

```
describe('When a POST measure arrives with a TimeInstant query parameter in the body with explicitAttrs:false', function() {
```

Reply:
Fixed in 4724c32